scripts/rspec/rspec.ps1

```diff
@@ -36,14 +36,17 @@ $ErrorActionPreference = "Stop"
 $RuleTemplateFolder = "${PSScriptRoot}\\rspec-templates"
 
 # Update the following variable when a new version of rule-api has to be used.
-$rule_api_version = "1.24.2.1313"
-$rule_api_error = "Download Rule-api from " + `
-    "'https://repox.jfrog.io/repox/sonarsource-private-releases/com/sonarsource/rule-api/rule-api/${rule_api_version}' " +`
+$rule_api_error = "Could not find the Rule API Jar locally. Please download the latest rule-api from " + `
```

Comment:
ideally we'd download it, but we need to authenticate and I'm not sure it's worth the effort